Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -559,6 +559,10 @@
 workflows[142.901] = ['',['RunUPC2023','RECODR3_2024_UPC','HARVESTDPROMPTR3']]
 workflows[142.902] = ['',['RunUPC2023','RECODR3_2024_HIN','HARVESTDPROMPTR3']]
 
+### run3-2024 skim (2024 HI MC temp)
+workflows[143.201] = ['',['HydjetQ_B12_5362GeV_2024','DIGIHI2024','SKIMHIFORWARDRUN3_2024','HARVESTHI2024']]
+workflows[143.202] = ['',['HydjetQ_B12_5362GeV_2024','DIGIHI2024','SKIMHIPHYSICSRAWPRIMERUN3_2024','HARVESTHI2024']]
```

Comment:
hmm, I think the rawprime wf should use the rawprime reco. 
see 
https://github.com/cms-sw/cmssw/blob/15283d2354f36562a6adc97318849ece08035b5b/Configuration/PyReleaseValidation/python/relval_standard.py#L874

Reply:
OK, let's follow this wf.

Reply:
Well,  if you run `SKIMHIPHYSICSRAWPRIMERUN3_2024` as defined right now, you are not using the strip approximate cluster modifier used in `RECOHI2024PROD`:

https://github.com/cms-sw/cmssw/blob/ce4d0b1395f39ad6ea81b98cd4dab57c3ac4df60/Configuration/PyReleaseValidation/python/relval_steps.py#L3261-L3266

then's pointless to run the `RAWPRIMESIMHI18` step in between as you'll just consume the regular raw and not rawprime.

Reply:
My bad. Now it's added. If it's correct, please test.